### DISCUSSION TITLE
[Verticapy 0.12.0] SQL magic -nrows and -ncols options

### DISCUSSION THREAD

**Discussion by rbensa** [vertica/VerticaPy#510] [Q&A] [answered]
Hello,

There is currently a problem using the -nrows/ncols options, the set_option uses a default value for the numbers of rows and columns, this option takes precedence over the two options previously mentioned.

Kind regards,

Réda

**Comment by oualib** [comment, marked as answer]
Hi Réda, it should not. If you look at the sql_magic file, you can see that the vDataFrame has its own limits. I've tried it and it is working perfectly.
